### PR TITLE
solve #98 footer is not responsive issue

### LINKED ISSUE #98: footer is not responsive
**Describe the bug**
The content of footer is overflow.

**To Reproduce**
1. can use media query OR border box property.

**Expected behavior**
It should not be overflowed.

**Screenshots**
footerbug

### PR DESCRIPTION
**Pull request by Vishal-raj-1**
### Issue Fix
Fixes #98 

**Description of changes made** 
Earlier it was overflowed as mentioned it #98 . for making it responsive what I have done is, I have adding the background color to footer so that it will not overflow now.

**Submissions guide:**

- [ ] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [ ] Have you lint your code locally prior to submission?

@s-ayush2903 @Ankit7Das Please look at this PR after merging this #102 one!!

### CODE REVIEW

**Review by s-ayush2903 — CHANGES_REQUESTED** [review]
For the issue this PR targets, we need only the latest commit of this PR as of now, right now you have very old commits from your old branches, you should delete 'em and force push the branch here. Here, you'll need to use `reset --hard HEAD` for deleting irrelevant commits in this branch, although this is a very dangerous command, but this'll do the trick needed. Read about it [here](https://git-scm.com/docs/git-reset)

File: contact.html [hunk 1]
```diff
@@ -56,6 +56,7 @@
         $('.nwoc-drawer-item').click(function() {
           $( '.mdl-layout__drawer, .mdl-layout__obfuscator' ).removeClass( 'is-visible' );
         });
+      });
```
Comment (Vishal-raj-1) [inline contact.html:59]: these braces are missing. I have debug that issue from my console.

**Comment by Vishal-raj-1** [comment]
> For the issue this PR targets, we need only the latest commit of this PR as of now, right now you have very old commits from your old branches, you should delete 'em and force push the branch here. Here, you'll need to use `reset --hard HEAD` for deleting irrelevant commits in this branch, although this is a very dangerous command, but this'll do the trick needed. Read about it [here](https://git-scm.com/docs/git-reset)

No no Sir !! I know this commits message is shown till my branch  is not updated with master branch of this repo. doing `git pull upstream master` and `git push` works here. 

@s-ayush2903 this is fine now !! check it

**Review by s-ayush2903 — CHANGES_REQUESTED** [review]
The text now overlaps with the back to top button, this should be fixed

**Comment by Vishal-raj-1** [comment]
> The text now overlaps with the back to top button, this should be fixed

It's done.

**Comment by Vishal-raj-1** [comment]
> It's done.

Any Updates @s-ayush2903 ??

**Comment by s-ayush2903** [comment]
Ah, it is clean, will merge it now. Was busy in college these days, thanks for pinging :+1:

**Comment by Vishal-raj-1** [comment]
> Ah, it is clean, will merge it now. Was busy in college these days, thanks for pinging 👍

No Problem !!

**Review by s-ayush2903 — APPROVED** [review]
LGTM

**Pull request merged**